src/components/ToggleBlock/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export default function ToggleBlock({
+  title,
+  children,
+  headingClasses,
+  innerClasses,
+  allowToggle,
+}) {
+  const [showFacets, toggleShowFacets] = useState(true);
+  let facetBlockHeading = <h2 className={headingClasses}>{title}</h2>;
+  if (allowToggle) {
+    facetBlockHeading = (
+      <h2 className={headingClasses}>
+        <button type="button" onClick={() => toggleShowFacets(!showFacets)}>
+          {title}
+        </button>
+      </h2>
+    );
+  }
+  return (
+    <section>
+      {facetBlockHeading}
+      {showFacets
+        && (
+          <div className={innerClasses}>
+            {children}
+          </div>
+        )}
+    </section>
+  );
+}
+
+ToggleBlock.defaultProps = {
+  allowToggle: true,
+  headingClasses: 'dkan-facet-block-title',
```

Comment:
We want to avoid specific catalog branding, can you remove the 'dkan-' on the default classes?